src/dbnode/server/server.go

```diff
@@ -599,6 +600,10 @@ func Run(runOpts RunOptions) {
 		// SetDatabase() once we've initialized it.
 		service = ttnode.NewService(nil, ttopts)
 	)
+	if cfg.Tchannel.MaxIdleTime > 0 && cfg.Tchannel.IdleCheckInterval > 0 {
+		tchannelOpts.MaxIdleTime = cfg.Tchannel.MaxIdleTime
+		tchannelOpts.IdleCheckInterval = cfg.Tchannel.IdleCheckInterval
```

Comment:
nit: Does this need to check that `tchannelOpts.MaxIdleTime >= tchannelOpts.IdleCheckInterval`?

Reply:
The check interval can be independent of the max idle time.